src/dropdown/list/dropdown-list.component.ts

```diff
@@ -292,6 +299,7 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnDest
 			if (this.displayItems) {
 				this.index = 0;
 			}
+			this.changeDetectorRef.markForCheck();
```

Comment:
Added the `markForCheck()` here because the component `ComboBox` uses `DropdownList` and there is a unit test that checks "should display dropdown list when new items are passed that match the current search string".